src/Components/Imaging/PatientAdmission/PatientDetailsBlock/index.js

```diff
@@ -197,60 +205,94 @@ const PatientDetailsBlock = ({
             );
           }
         }
+        let item = [];
+        if (configuration.clinikal_pa_commitment_form === '1') {
```

Comment:
according the requirement if clinikal_pa_commitment_form is true need to show 2 tabs of commitment and payment both.
seems in the code it not walk together